pkgs/by-name/pi/pike/package.nix

```diff
@@ -0,0 +1,196 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  pkg-config,
+  autoconf,
+  automake,
+  libtool,
+  bison,
+  flex,
+  gmp,
+  pcre,
+  nettle,
+  libjpeg,
+  libpng,
+  libtiff,
+  freetype,
+  sqlite,
+  libmysqlclient,
+  postgresql,
+  openssl,
+  zlib,
+  bzip2,
+  gdbm,
+  libffi,
+  expat,
+  libxml2,
+  libxslt,
+  gtk3,
+  pango,
+  cairo,
+  glib,
+  SDL2,
+  libGL,
+  dpkg,
+  autoPatchelfHook,
+  ncurses,
+  libnsl,
+  libxcrypt-legacy,
+  unstableGitUpdater,
+}:
+
+let
+  # Pre-built Pike binary from Debian for bootstrapping
+  pike-bootstrap = stdenv.mkDerivation rec {
+    pname = "pike-bootstrap";
+    version = "8.0.1956";
```

Comment:
Given this version number, should the commit message not be "pike: init at 8.0.1956"?